android-components/components/browser/state/src/main/java/mozilla/components/browser/state/action/BrowserAction.kt

```diff
@@ -946,6 +958,17 @@ sealed class TranslationsAction : BrowserAction() {
         override val tabId: String,
         val supportedLanguages: TranslationSupport?,
     ) : TranslationsAction(), ActionWithTab
+
+    /**
+     * Sets the given page settings on the page on the given [tabId]'s store.
+     *
+     * @property tabId The ID of the tab the [EngineSession] should be linked to.
+     * @property pageSettings The new page settings.
+     */
+    data class TranslateSetSettingsAction(
+        override val tabId: String,
+        val pageSettings: TranslationPageSettings?,
+    ) : TranslationsAction(), ActionWithTab
```

Comment:
Yeah, totally agree - I feel like some of these actions were getting especially wordy and long with both the prefix and postfix. 

I filed [bug 1877523](https://bugzilla.mozilla.org/show_bug.cgi?id=1877523), so it is  noted that the naming can get out of hand pretty quickly with this pattern.

Reply:
Thought about this some more, I think I want to establish a new pattern starting here to remove the `Translate` part on these two new actions because they are becoming too wordy.

Now:
* `OperationRequestedAction`
* `SetSettingsAction`

Will refactor existing actions when it seems naturally applicable/relevant/safe to do so on a patch.